modules/37_industry/subsectors/equations.gms

```diff
@@ -13,26 +13,36 @@
 ***------------------------------------------------------
 *' Industry final energy balance
 ***------------------------------------------------------
-q37_demFeIndst(t,regi,entyFe,emiMkt)$( entyFe2Sector(entyFe,"indst") ) ..
-  sum(se2fe(entySe,entyFe,te),
-    vm_demFeSector_afterTax(t,regi,entySe,entyFe,"indst",emiMkt)
+q37_demFeIndst_intermediate(t,regi,in,secInd37,emiMkt)$(
+                                          ppfen_industry_dyn37(in)
+                                      AND secInd37_2_pf(secInd37,in)
+                                      AND secInd37_emiMkt(secInd37,emiMkt) ) ..
+  sum((sefe(entySe,entyFe),
+       fe2ppfEn(entyFe,in)),
+    v37_demFeIndst(t,regi,entySe,entyFe,in,emiMkt)
```

Comment:
That's actually quite a fundamental problem: `ppfen` do not exist any more for process-based sectors, so the whole variable definition (and thus the FE balance) is invalid for them. 
Could you maybe define `v37_demFeIndst(t,regi,entySe,entyFe,secInd37,emiMkt)` instead? That would get you the full information, apart from the information of primary vs secondary steel. Otherwise, you'd have to skip that intermediate step for pbs - I don't know what implications that would have for reporting. 
Sorry about that.

Reply:
Bottom line: PR needs significant changes in order to not break the process-based implementation. @LaviniaBaumstark